src/webviews/extension-server/WebviewController.ts

```diff
@@ -62,6 +63,14 @@ export class WebviewController<Configuration> extends WebviewBaseController<Conf
                     case 'confirm':
                         SharedState.itemsToClear = message.itemsToClear ?? 0;
                         this._panel.dispose();
+                        break;
+                    case 'copilot':
+                        await vscode.commands.executeCommand("workbench.action.chat.newChat");
+                        await vscode.commands.executeCommand("workbench.action.chat.open", {
```

Comment:
I was running into an issue where if the chat wasn't already open, `newChat` wouldn't actually start a new chat.  Do you run into this issue at all?  If so, I was calling open before `newChat` to fix this behavior, see:

https://github.com/microsoft/vscode-azureresourcegroups/blob/main/src/chat/askAgentAboutActivityLog.ts